src/prompts/index.tsx

```diff
@@ -267,22 +340,24 @@ export class PromptManager {
                 this.subscriptionState = await this.pushOpsManager?.getCurrentSubscriptionState(
                     this.context
                 );
-                this.evaluateTriggers();
+                await this.evaluateTriggers();
                 this.render();
                 break;
         }
     }
 
     private async loadPrompts(): Promise<void> {
+        this.platformConfig = await loadConfig(this.context);
```

Comment:
There was a follow-up bug about error handling inside loadConfig. It doesn't seem to respect server errors as errors, and leads to it caching nonsense into the config slot.

Are we picking that up here / is it going to be a follow-up?

Just cautious we should be try/catch if it can throw. Actually, did you already handle this w/ `PlatformConfig | null` return?

Reply:
Ahh yes was aware of the bug and wanted to pick this up a separate focus - there were try/catch blocks around this call too, to an extent, but the catch within the load op was a little pokemon, catching and swallowing them up too - initial idea was to rethrow/propagate but would need to see what that breaks and start from there etc.